.github/workflows/python-package.yml

```diff
@@ -28,4 +28,9 @@ jobs:
         flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest
       run: |
-        echo Add your testing here
+        pytest -v
```

Comment:
Nice.
A question: how do we save test results to a file and send to Slack?

Reply:
pytest -v >> test_results.txt